packages/experiments-realm/garden-app.gts

```diff
@@ -1,4 +1,5 @@
-import { AppCard } from './app-card';
+// @ts-expect-error: Module '/catalog/app-card' may not be available during compilation
+import { AppCard } from '/catalog/app-card';
```

Comment:
Can we do catalog imports from that package the way we import from other packages? `@cardstack/catalog-realm`? we may need to adjust some config so that it's possible

Reply:
<img width="459" alt="tsconfig" src="https://github.com/user-attachments/assets/e74f21f3-6c93-4b73-8f10-7ef71694423e">
Maybe something like this in the tsconfig. There might be more to it.

Reply:
hmmmm, that's a good point. @lukemelia this might be an argument for a static URL like the base realm so that code that is written in any env can import catalog realm modules